applications/machine_learning/README.rst

```diff
@@ -79,6 +79,31 @@ By default, the application uses pretrained machine leaning models deployed in `
 The application displays LED effects that correspond to the machine learning results.
 For more detailed information, see the `User interface`_ section.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the Machine Learning application specified modules can be suspended or turned off if they are not in use.
+The application uses :ref:`caf_power_manager` module for that purpose.
+It means that Zephyr power management is used when device is in active on suspended state,
+but power off state is forced directly by :ref:`caf_power_manager`.
+
+* In the :c:enumerator:`POWER_MANAGER_LEVEL_SUSPENDED` state, the device maintains the active Bluetooth connection.
+* In the powered off state, the CPU is switched to the off mode.
+
+In both cases, most of the functionalities are disabled.
+For example, LEDs and sensors are turned off and advertising is stopped.
```

Comment:
Thingy:53's sensor is not turned off actually (it is waiting for trigger). Maybe we should not mention sensors here?

Reply:
I will change it, but please note that this one is direct copy from nrf_desktop.